src/scripts/components/table/table.js

```diff
@@ -27,18 +28,18 @@ export class Table extends Component {
   updateTableHead(date) {
     this.currentDate = date;
 
-    this.month = Number.parseInt(this._currentDate.toLocaleDateString('en-US', { month: 'numeric'}));
-    this.year = Number.parseInt(this._currentDate.toLocaleDateString('en-US', { year: 'numeric'}));
+    this.month = Number.parseInt(this._currentDate.toLocaleDateString('en-US', {month: 'numeric'}));
```

Comment:
Why this was changed? This changes is not realted to the modal window, fix this, please.